easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018a-Python-2.7.14-HDF5-1.10.1.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.3'
+hdf5_ver = '1.10.1'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s' % hdf5_ver
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+#checksums = ['eb25d6ee85f4f5ac1d5581958f8c6eed9b1d50746f82866fe92e507541def35b']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('netCDF', '4.6.0', '-HDF5-%s' % hdf5_ver),
+    ('expat', '2.2.5'),
+    ('GEOS', '3.6.2', '-Python-%(pyver)s'),
+    ('SQLite', '3.20.1'),
+    ('libxml2', '2.9.7'),
+    ('libpng', '1.6.34'),
+    ('libjpeg-turbo', '1.5.2'),
+    ('JasPer', '1.900.1'),
+    ('LibTIFF', '4.0.9'),
+    ('zlib', '1.2.11'),
+    ('cURL', '7.55.1'),
+    ('PCRE', '8.41'),
```

Comment:
`PROJ` dependency is missing, see https://github.com/easybuilders/easybuild-easyconfigs/pull/5438/files#diff-dae1edc29c65d6a8ea5ded8b7f2b4650R34